ChangeLog

```diff
@@ -1,4 +1,8 @@
 2017-07-24  Leah South  <leah.south@hdr.qut.edu.au>
+
+    * inst/include/sampler.h: Correcting the description of GetParticleWeight().
+    
+2017-07-24  Leah South  <leah.south@hdr.qut.edu.au>
```

Comment:
Sorry are you referring to merging the entries in the ChangeLog or another type of merge? I just did a commit to remove the unnecessary ChangeLog entry.